.editorconfig

```diff
@@ -19,6 +19,9 @@ indent_size = 2
 [*.jsx]
 indent_size = 2
 
+[*.td]
+indent_size = 2
+
```

Comment:
personally i would break this commit out to a different pr...ill stamp that one tho!

Reply:
Reverted!